plugin/PDFContextMenu.qml

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2016 Caliste Damien.
+ * Contact: Damien Caliste <dcaliste@free.fr>
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; version 2 only.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+
+Rectangle {
+    id: root
+
+    property alias url: labelUrl.text
+
+    gradient: Gradient {
+        GradientStop { position: 0.0; color: Theme.highlightDimmerColor }
+        GradientStop { position: 1.0; color: Theme.rgba(Theme.highlightDimmerColor, .8) }
+    }
+    opacity: 0.
+    Behavior on opacity { FadeAnimation { duration: 300 } }
+    visible: opacity > 0.
+    anchors.fill: parent
+        
+    MouseArea {
+        anchors.fill: parent
+        onClicked: root.opacity = 0.
+        preventStealing: true
+    }
+
+    Column {
+        spacing: Theme.paddingMedium
+        anchors.top: parent.top
+        anchors.topMargin: Theme.paddingLarge
+        width: parent.width
+        InfoLabel {
+            font.pixelSize: Theme.fontSizeLarge
+            //% "External link"
+            text: qsTrId("sailfish-office-tl-pdf-link")
+            color: Theme.highlightColor
+        }
+        InfoLabel {
+            id: labelUrl
+            font.pixelSize: Theme.fontSizeMedium
+            text: url
+            color: Theme.secondaryHighlightColor
+            wrapMode: Text.WrapAtWordBoundaryOrAnywhere
+        }
+    }
+
+    Column {
+        id: menu
+
+        anchors.bottom: parent.bottom
+        anchors.bottomMargin: Theme.paddingLarge
+        width: parent.width
+
+        BackgroundItem {
+            MenuItem {
+                anchors.horizontalCenter: parent.horizontalCenter
+                //% "Open link in browser"
+                text: qsTrId("sailfish-office-me-pdf-open-link")
```

Comment:
You're right, I guess so. I'll generate a PDF like that to test and adapt the UI accordingly.